bin/egencache

```diff
@@ -1105,12 +1105,11 @@ try:
         if "PATH" in os.environ:
             env["PATH"] = os.environ["PATH"]
 
-        if not sys.stdout.isatty() or os.environ.get("NOCOLOR", "").lower() in (
-            "yes",
-            "true",
+        if not sys.stdout.isatty() or portage.util.no_color(
+            os.environ.get("NOCOLOR", "false").lower(), os.environ.get("NO_COLOR")
```

Comment:
Probably I am missing something, but couldn't the two `os.environ.get` calls be moved into the `no_color` function?

Reply:
but then we wouldn't be able to use it with portage.settings.get or settings.get

Reply:
I see. I thought that I was missing something. :smile: